### PR TITLE
fix #1695 add demo & doc

### LINKED ISSUE #1695: [antd 1.0.1]Select组件onChange的第二参数总是undefined
```js
handleChange(selectKeys, labels) {
    console.log(selectKeys, labels);
}
```

```html
<Select {...otherProps}
              multiple
              showSearch searchPlaceholder="模糊搜索"
              placeholder="选择方案配置, 最多两个"
              optionFilterProp="children"
              onChange={this.handleChange}
              notFoundContent="获取中..."
              value={this.props.value}>
  {this.state.data.map(record => <Select.Option key={record.id} value={record.sid}>{`${record.sid} - `}{record.taskName}</Select.Option>)}
</Select>
```

> otherProps是form传入的一些属性，onChange在之后又一次赋值，应该不是form的onChange

console打印：

```
[1005] undefined
```

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [ ] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [ ] Rebase before creating a PR to keep commit history clear.
- [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: components/select/demo/label-in-value.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+order: 10
+title: 获取label
```
Comment (superRaytin) [inline components/select/demo/label-in-value.md]: 中英文之间加空格~

File: components/select/index.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ english: Select
...
+| value    | 指定当前选中的条目 | String/Array<String>/{key: String, label: React.Node}/Array<{key, label}>   |  无  |
+| defaultValue | 指定默认选中的条目 | String/Array<String>/{key: String, label: React.Node}/Array<{key, label}>   |  无  |
 | multiple   | 支持多选 | boolean | false |
 | allowClear   | 支持清除, 单选模式有效 | boolean | false |
 | filterOption | 是否根据输入项进行筛选。当其为一个函数时，会接收 `inputValue` `option` 两个参数，当 `option` 符合筛选条件时，应返回 `true`，反之则返回 `false`。 | boolean or function(inputValue, option) | true     |
 | tags | 可以把随意输入的条目作为 tag，输入项不需要与下拉选项匹配 | boolean |false |
 | onSelect | 被选中时调用，参数为选中项的 value 值 | function(value, option) | 无   |
 | onDeselect | 取消选中时调用，参数为选中项的 option value 值，仅在 multiple 或 tags 模式下生效 |  function(value) | 无   |
-| onChange | 选中option，或input的value变化(combobox 模式下)时，调用此函数 | function(value, label) | 无 |
+| onChange | 选中option，或input的value变化(combobox 模式下)时，调用此函数 | function(value) | 无 |
```
Comment (superRaytin) [inline components/select/index.md]: 同上

Comment (superRaytin) [inline components/select/index.md]: 中英混输的时候 () 用全角符号~

**Comment by superRaytin** [comment]
commit `fix#1695 add demo & doc` 需要改为 `fix #1695 ...` 这种，这样可以链接到对应 issue

**Comment by ddcat1115** [comment]
调整好啦~

**Comment by superRaytin** [comment]
👌

**Pull request merged**